src/editor/main.cpp

```diff
@@ -225,7 +225,7 @@ void initialize_webview(HWND hwnd, int nCmdShow) {
 
         g_webview.NewWindowRequested([=](IWebViewControl sender_requester, WebViewControlNewWindowRequestedEventArgs args) {
           // Open the requested link in the default browser registered in the Shell
-          int res = (int)ShellExecute(nullptr, L"open", args.Uri().AbsoluteUri().c_str(), nullptr, nullptr, SW_SHOWNORMAL);
+          int res = static_cast<int>(reinterpret_cast<uintptr_t>(ShellExecute(nullptr, L"open", args.Uri().AbsoluteUri().c_str(), nullptr, nullptr, SW_SHOWNORMAL)));
```

Comment:
this can probably be written as `auto res = reinterpret_cast<uintptr_t>(ShellExecute(...));`